test/fakeintake/server/server_test.go

```diff
@@ -118,6 +120,43 @@ func TestServer(t *testing.T) {
 		assert.Equal(t, expectedGETResponse, actualGETResponse, "unexpected GET response")
 	})
 
+	t.Run("should store multiple payloads on any route and return them in json", func(t *testing.T) {
+		clock := clock.NewMock()
+		fi := NewServer(WithClock(clock))
+
+		postSomePayloads(t, fi)
+
+		request, err := http.NewRequest(http.MethodGet, "/fakeintake/payloads?endpoint=/api/v2/logs&format=json", nil)
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
TODO later - can integrate in my PR - we should test that on `format=raw` we send raw payloads (spoiler it doesn't, but there is a fix on my PR)